Makefile

```diff
@@ -59,6 +59,11 @@ install-sylius:
 	${YARN} build
 	${CONSOLE} cache:clear
 
+configure-sylius:
+	cd ${TEST_DIRECTORY} && echo '    Synolia\SyliusSchedulerCommandPlugin\Checker\SoftLimitThresholdIsDueChecker:' >> config/services.yaml
+	cd ${TEST_DIRECTORY} && echo '        tags:' >> config/services.yaml
+	cd ${TEST_DIRECTORY} && echo '            - { name: !php/const Synolia\SyliusSchedulerCommandPlugin\Checker\IsDueCheckerInterface::TAG_ID }' >> config/services.yaml
```

Comment:
A file should be created in tests/data/config/ with this data and he will be automaticly copied.
Example : https://github.com/synolia/SyliusSchedulerCommandPlugin/blob/master/tests/data/config/packages/test/sylius_fixtures.yaml

Reply:
My bad, in defautl Kernel files under config/services/* hare not loaded